robber/matchers/equal.py

```diff
@@ -5,6 +5,13 @@
 from robber.helper import unicode_to_str
 from robber.matchers.base import Base
 
+try:
+    from collections import OrderedDict
+except ImportError:
+    ordered_dict_available = False
+else:
+    ordered_dict_available = True
```

Comment:
https://docs.python.org/2/library/collections.html 
According to this, OrderedDict is new in Python 2.7, but we support Python 2.6, so...